frontend/src/libs/components/profile-settings-sidebar/libs/constants.ts

```diff
@@ -0,0 +1,10 @@
+import { type SettingsOption } from '#libs/types/settings-options.type.js';
+
+const settingsOptions: SettingsOption[] = [
```

Comment:
Follow quality criteria

Reply:
@v0ldemar01 just to clarify:
Are we talking about  B22. Constant names are written in capital letters ? 
I was not sure that I need to apply it in this case, since I was following general approach of chat sidebar that had its array with options in lowercase.

Reply:
So it's a separate constant that need to be normalised with B22 rule

Reply:
You can check other places of constants.ts to clarify about it

Reply:
Understood, working on it